### PR TITLE
Add readme template

### LINKED ISSUE #44: Template for package READMEs
### Feature request

Create a "template" file to use for all package readmes using special global variables that are replaced on compile time. This should help normalize some of the extra documentation that goes with all packages.

### Example `template.md`:

````md
%DESCRIPTION%

![NPM Version](https://img.shields.io/npm/v/%NAME%)
![NPM Downloads](https://img.shields.io/npm/dt/%NAME%)

Last published %DATE%

## Installation

`%NAME%` can be installed...

```shell
npm i %NAME%
```

<!-- this is where the individual README.md content goes -->

%README%

## Contribute

...

````

### Future work:

Example programs should be committed to this repository and run during the `npm test` phase but not published to npm. This could even further automate the documentation by adding a `%EXAMPLE%` super global variable that is replaced by the full example program on compile time.

### LINKED ISSUE #45: Display LaTeX equations in documentation
Looking at https://www.mathjax.org/

### PR DESCRIPTION
**Pull request by nicfv**
- Closes #44 
- Closes #45

### CODE REVIEW

**Comment by nicfv** [comment]
Included MathJax support. Will now close #45 as well.

**Comment by nicfv** [comment]
Global variables:
- NAME
- DESC
- DIR
- VER
- TIME

Only DESC and DIR can possibly contain `:`
